### DISCUSSION TITLE
Netflix player does not work

### DISCUSSION THREAD

**Discussion by DjCristianquezQuez** [AndreyPavlenko/Fermata#131] [General]
Hii 

Fisrt i hace yo congrats for the app 

The Netflix player does not working

Página web no disponible
No ha sido posible cargar la página web intent://www.netflix.com/title/80014749#Intent;scheme=https;package=com.netflix.mediaclient;S.browser_fallback_url=https%3A%2F%2Fwww.netflix.com%2Ftitle%2F80014749%3FpreventIntent%3Dtrue;end porque:

net::ERR_UNKNOWN_URL_SCHEME

**Comment by AndreyPavlenko** [comment]
Have you tried to enable the "Desktop version" mode? You may also try to change the user agent string.

**Comment by DjCristianquezQuez** [comment]
I Will trie thiss thanks

**Comment by QB116** [comment]
I've tried changing the user agent strings on my android but still haven't had any luck. Any other ideas. Thanks in advance8

**Comment by DjCristianquezQuez** [comment]
yo ahora estoy teniendo problemas con android auto, no reconoce ciertas
aplicaciones. una de ellas es fermata cuando lo solucione te lo reviso
a que te refieres con  las cadenas del agente de usuario

**Comment by AndreyPavlenko** [comment]
Seems, currently it's not possible to add Netflix support: https://stackoverflow.com/questions/35285671/netflix-in-android-webview

**Comment by madala1819** [comment]
Can any one please suggest me how to install or run Netflix and prime on android auto
